requirements.txt

```diff
@@ -5,5 +5,7 @@ django_compressor==2.1
 djmail==0.11.0
 futures==3.0.3
 premailer==2.9.6
-psycopg2==2.6.1
+psycopg2==2.7.3
 requests==2.9.1
+gunicorn
+whitenoise
```

Comment:
How is this? A `pip freeze` always gives the package version

Reply:
Noted, have updated, thank you